climakitae/core/data_export.py

```diff
@@ -225,13 +228,14 @@ def _convert_da_to_ds(data: xr.DataArray | xr.Dataset) -> xr.Dataset:
     ----------
     data: xarray.DataArray or xarray.Dataset
     """
-    if isinstance(data, xr.core.dataarray.DataArray):
-        if not data.name:
-            # name it in order to call to_dataset on it
-            data.name = "data"
-        return data.to_dataset()
-    elif isinstance(data, xr.core.dataset.Dataset):
-        return data
+    match data:
+        case data if isinstance(data, xr.core.dataarray.DataArray):
+            if not data.name:
```

Comment:
formatting

Reply:
Fixed in [f167843](https://github.com/cal-adapt/climakitae/pull/564/commits/f1678435f3eb4e93416a32603d7b0aa3c710a3a7)